.circleci/config.yml

```diff
@@ -323,6 +343,14 @@ workflows:
           requires:
             - py3_dagmc_pymoab_build
 
+      - py3_dagmc_pymoab_hdf5_12_build:
```

Comment:
@bam241 How do I make it so that this only produces a warning - and not a failure?

Reply:
Or is that a Travis thing?

Reply:
I don't think this exists in CircleCI.

but we can decide to block or not merge based on the results of some tests.